include/tallies/zernike_polynomial.hpp

```diff
@@ -23,6 +23,9 @@ class ZernikePolynomials {
   std::vector<double> evaluate_zernikes(const double r,
                                         const double theta) const;
 
+  double orthonormalization_constant(const std::size_t& order,
+                                     bool is_pi_multiply = true) const;
```

Comment:
Maybe rename the `is_pi_multiply` variable. It is not clear to me whether this means we need it multiplied by pi or not. Also, is this option even necessary ? We should try to keep the data as consistent as possible and not allow such a variation if possible.

Reply:
I have realised that it can create confusion. So, I have removed it.